src/site/content/en/blog/bluetooth/index.md

```diff
@@ -0,0 +1,549 @@
+to [request] and [connect to] nearby Bluetooth Low Energy devices,
+[read]/[write] Bluetooth characteristics, [receive GATT Notifications], know
+when a [Bluetooth device gets disconnected], and even [read and write to
+Bluetooth descriptors].
+
+For Linux and earlier versions of Windows, enable the
+`#experimental-web-platform-features` flag in `chrome://flags`.
+
+### Available for Origin Trials
+
+In order to get as much feedback as possible from developers using the Web
+Bluetooth API in the field, we've previously added this feature in Chrome 53 as
+an [origin trial] for Chrome OS, Android, and Mac.
+
+The trial has successfully ended in January 2017.
+
+## Security Requirements
+
+To understand the security tradeoffs, I recommend the [Web Bluetooth Security
+Model] post from Jeffrey Yasskin, a software engineer on the Chrome team,
+working on the Web Bluetooth API specification.
+
+### HTTPS Only
+
+Because this experimental API is a powerful new feature added to the Web, it is
+made available only to [secure contexts]. This means you'll need to build with
+TLS in mind.
+
+### User Gesture Required
+
+As a security feature, discovering Bluetooth devices with
+`navigator.bluetooth.requestDevice` must be triggered by [a user gesture] such
+as a touch or a mouse click. We're talking about listening to
+[`pointerup`], `click`, and `touchend` events.
+
+```js
+button.addEventListener('pointerup', function(event) {
+  // Call navigator.bluetooth.requestDevice
+});
+```
+
+## Get into the Code
+
+The Web Bluetooth API relies heavily on JavaScript [Promises]. If you're not
+familiar with them, check out this great [Promises tutorial]. One more thing,
+`() => {}` are simply ECMAScript 2015 [Arrow functions] -- they have a shorter
+syntax compared to function expressions and lexically bind the `this` value.
+
+### Request Bluetooth Devices
+
+This version of the Web Bluetooth API specification allows websites, running in
+the Central role, to connect to remote GATT Servers over a BLE connection. It
+supports communication among devices that implement Bluetooth 4.0 or later.
+
+When a website requests access to nearby devices using
+`navigator.bluetooth.requestDevice`, the browser will prompt user with a device
+chooser where they can pick one device or simply cancel the request.
+
+
+<figure class="w-figure">
+  <video controls autoplay loop muted>
+    <source src="https://storage.googleapis.com/web-dev-assets/bluetooth/bluetooth-device-chooser.mp4">
+  </video>
+  <figcaption class="w-figcaption">Bluetooth device user prompt.</figcaption>
+</figure>
+
+
+The `navigator.bluetooth.requestDevice` function takes a mandatory Object that
+defines filters. These filters are used to return only devices that match some
+advertised Bluetooth GATT services and/or the device name.
+
+For instance, requesting Bluetooth devices advertising the [Bluetooth GATT
+Battery Service] is this simple:
```

Comment:
```suggestion
For instance, to request Bluetooth devices advertising the [Bluetooth GATT
Battery Service]:
```
I try to avoid labeling things as simple or easy and just stick to the facts. It's not a big deal here and I think one could argue that this is indeed simple, but for the sake of consistency throughout the web.dev codebase I'd prefer to continue avoiding it.

Reply:
Done.